<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseWidget</class>
 <widget class="QWidget" name="ChooseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>请选择要出战的精灵</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="boxMyPoke"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>选择目标精灵</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="boxEnemyPoke">
        <item>
         <property name="text">
          <string>0级力量系</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5级肉盾系</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10级防御系</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15级敏捷系</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="buttonDecide">
     <property name="text">
      <string>决定！</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
